### PR TITLE
feat(dedicated.nutanix): update of label and link in tile

### PR DESCRIPTION
**Pull request by dectotam**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/nutanix-package-v2
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Feat MANAGER-13945
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

## Description

Content update of label in tile on Nutanix package V2  : Pack Link & baie(s)

### CODE REVIEW

File: packages/manager/modules/nutanix/src/dashboard/general-info/constants.js [hunk 1]
```diff
@@ -3,44 +3,29 @@ export const TRAVAUX_LINK = {
...
-  NL: `${GUIDES_URL.NL}packaged`,
-  PL: `${GUIDES_URL.PL}packaged`,
-  PT: `${GUIDES_URL.PT}packaged`,
-  MA: `${GUIDES_URL.MA}packaged`,
-  SN: `${GUIDES_URL.SN}packaged`,
-  TN: `${GUIDES_URL.TN}packaged`,
-  WS: `${GUIDES_URL.WS}packaged`,
-  US: `${GUIDES_URL.US}packaged`,
-  DEFAULT: `${GUIDES_URL.DEFAULT}packaged`,
+export const GUIDE_PACKAGES_ENDPOINT = {
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/general-info/constants.js:26]: endpoint refers to an API usually so it's a bit confusing. 
```suggestion
export const GUIDE_PACKAGES_PATH = {
```

File: packages/manager/modules/nutanix/src/dashboard/general-info/constants.js [hunk 2]
```diff
@@ -3,44 +3,29 @@ export const TRAVAUX_LINK = {
...
+  BYOL: '/byol',
+  DEFAULT: '/packaged',
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/general-info/constants.js:28]: packaged are for pack offers so it's not really the default one. 
```suggestion
  PACK: '/packaged',
```

File: packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licence-tile-features/controller.js [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
+} from '../../../general-info/constants';
 
 export default class NutanixLicenceTileFeaturesCtrl {
   /* @ngInject */
   constructor(coreConfig) {
     const { ovhSubsidiary } = coreConfig.getUser();
-    this.guideLink = GUIDES_URL[ovhSubsidiary] || GUIDES_URL.DEFAULT;
+    this.guideLink =
+      (GUIDE_PACKAGES_BASE_URL[ovhSubsidiary] ||
+        GUIDE_PACKAGES_BASE_URL.DEFAULT) + GUIDE_PACKAGES_ENDPOINT.DEFAULT;
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licence-tile-features/controller.js:12]:
```suggestion
        GUIDE_PACKAGES_BASE_URL.DEFAULT) + GUIDE_PACKAGES_PATH.PACK;
```

File: packages/manager/modules/nutanix/src/dashboard/general-info/controller.js [hunk 4]
```diff
@@ -5,8 +5,10 @@ import {
   PRIVATE_BANDWIDTH_SERVICE_PREFIX,
   REPLICATION_FACTOR_PREFIX,
   TRAVAUX_LINK,
-  GUIDE_PACKAGES_URL,
+  GUIDE_PACKAGES_BASE_URL,
+  GUIDE_PACKAGES_ENDPOINT,
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/general-info/controller.js:9]:
```suggestion
  GUIDE_PACKAGES_PATH,
```

File: packages/manager/modules/nutanix/src/dashboard/general-info/controller.js [hunk 5]
```diff
@@ -22,21 +24,25 @@ export default class NutanixGeneralInfoCtrl {
...
   $onInit() {
     this.loadServicesDetails();
     this.setPrivateBandwidthServiceId();
     this.clusterRedeploying = this.cluster.status === CLUSTER_STATUS.DEPLOYING;
     this.showRedeployWarningModal = false;
+    this.NUTANIX_LINK =
+      (GUIDE_PACKAGES_BASE_URL[this.coreConfig.getUser().ovhSubsidiary] ||
+        GUIDE_PACKAGES_BASE_URL.DEFAULT) +
+      (GUIDE_PACKAGES_ENDPOINT[PACK_TYPE[this.packType]] ||
+        GUIDE_PACKAGES_ENDPOINT.DEFAULT);
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/general-info/controller.js:45]:
```suggestion
      (GUIDE_PACKAGES_PATH[PACK_TYPE[this.packType]] ||
        GUIDE_PACKAGES_PATH.PACK);
```

**Review by antonymarion — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Pull request merged**